src/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+const Checkbox = ({ handleCheckboxClick }) => {
+  return <input type="checkbox" onClick={handleCheckboxClick}></input>
+}
```

Comment:
Siguiendo la convención del issue #6 , el prop debería ser onCheck, porque es el evento. El handler es lo que se asigna a esa prop. Acordate siempre de pasarle los props por si necesitás agregarle nuevas cosas como `className`. Por otro lado, siendo un input, el evento debería ser `onChange`, no `onClick`. También los inputs son etiquetas abiertas (no pueden contener nada adentro).

```suggestion
const Checkbox = ({ onCheck, ...props }) => {
  return <input type="checkbox" onChange={onCheck} {...props} />
}
```